client/src/components/ProfilePhoto/ProfilePhoto.tsx

```diff
@@ -0,0 +1,33 @@
+import Grid from '@material-ui/core/Grid';
+import CircularProgress from '@material-ui/core/CircularProgress';
+import { useAuth } from '../../context/useAuthContext';
+import { useHistory } from 'react-router-dom';
+import { Paper, Typography, CssBaseline } from '@material-ui/core';
+import ProfilePhotoForm from './ProfilePhotoForm/ProfilePhotoForm';
+import useStyles from './useStyles';
+
+export default function Profile(): JSX.Element {
+  const classes = useStyles();
+  const { loggedInUser } = useAuth();
+
+  const history = useHistory();
+
+  if (loggedInUser === undefined) return <CircularProgress />;
+  if (!loggedInUser) {
```

Comment:
Are you expecting any other Falsy value here other than `undefined`?

Reply:
Deleted line 15